src/Umbraco.Web.UI.Client/src/views/propertyeditors/blocklist/umbBlockListPropertyEditor.component.js

```diff
@@ -176,6 +185,20 @@
 
             if (block === null) return null;
 
+            // ensure that the containing content variant language/culture is transfered along
+            // to the scaffolded content object representing this block. This is required for validation 
+            // along with ensuring that the umb-property inheritance is constently maintained.            
+            if (vm.umbVariantContent.editor.content.language) {
+                block.content.language = vm.umbVariantContent.editor.content.language;
+                // currently we only ever deal with invariant content for blocks so there's only one
+                block.content.variants[0].tabs.forEach(tab => {
+                    tab.properties.forEach(prop => {
+                        prop.culture = vm.umbVariantContent.editor.content.language.culture;
+                    });
+                });
+            }
+
+            // TODO: Why is there a '/' prefixed? that means this will never work with virtual directories
```

Comment:
This might be wrong, maybe it has to do with my setup. If you think its wrong, then let's make a PR with it corrected and have someone testing that it works. It does work, but that might be luck?

Reply:
Btw. there is multiple instances of similar appending.

Reply:
@nielslyngsoe It works currently for internal editors and will work for App_Plugin editors but we are 'rooting' the path. Websites aren't always hosted in the root path. Umbraco is possible to be hosted in a virtual folder like /cms/ instead of at / in which case this will never work because we are rooting this. This is why we have `umbRequestHelper.convertVirtualToAbsolutePath` and this code should use that instead. 

I need to move focus from 8.7 for now as we have other critical issues/support i need to work on. Are you able to create tasks for this?